### PR TITLE
[Organizations] Allow orgs to define the redirect URL after user regi…

### LINKED ISSUE #33201: [Organizations] Allow orgs to define the redirect URI after user registers or accepts invitation link
### Description

When a user follows an organization invitation or registration link, they are redirected to the account console upon completing the onboarding/registration. An organization should be able to defined the redirect URI that should be followed in these cases.

### Discussion

https://github.com/keycloak/keycloak/issues/30180#issuecomment-2311297651

### Motivation

Organizations should allow for customization of the redirect URIs after registration for a richer user experience.

### Details

_No response_

### PR DESCRIPTION
**Pull request by vramik**
…sters or accepts invitation link

Closes #33201

### CODE REVIEW

File: docs/documentation/server_admin/topics/organizations/managing-members.adoc [hunk 1]
```diff
@@ -86,7 +86,6 @@ intention to join the organization.
 On the other hand, if no user is associated with the given email address, the steps
 will involve creating a new account through the realm's self-registration flow. In this case, the user is forced
 to provide the same email address used to send the invitation.
-Make sure to enable the *User registration* setting in the *Login* tab at the *Realm settings* to use this type of invitation.
```
Comment (sguilhen) [inline docs/documentation/server_admin/topics/organizations/managing-members.adoc:89]: This change should be included in 26 regardless of what we do with this PR, right? If so I would open a separate PR to ensure we include this in the final documentation

File: docs/documentation/server_admin/topics/organizations/managing-organization.adoc [hunk 2]
```diff
@@ -11,6 +11,8 @@ To use organizations, you have to enable the feature for the current realm.
 
 . Toggle *Organizations* to *On*.
 
+. Click *Save*.
```
Comment (sguilhen) [inline docs/documentation/server_admin/topics/organizations/managing-organization.adoc:14]: Same thing for this one - should prob be part of a separate PR

File: docs/documentation/server_admin/topics/organizations/managing-members.adoc (hunk 1 above)
Comment (vramik) [reply docs/documentation/server_admin/topics/organizations/managing-members.adoc:89]: https://github.com/keycloak/keycloak/pull/33411

File: docs/documentation/server_admin/topics/organizations/managing-organization.adoc (hunk 2 above)
Comment (vramik) [reply docs/documentation/server_admin/topics/organizations/managing-organization.adoc:14]: https://github.com/keycloak/keycloak/pull/33411

**Review by pedroigor — APPROVED** [review]

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationInvitationLinkTest.java [hunk 3]
```diff
@@ -69,6 +72,11 @@ public class OrganizationInvitationLinkTest extends AbstractOrganizationTest {
     @Page
     protected RegisterPage registerPage;
 
+    @Before
+    public void setDriverTimeout() {
+        driver.manage().timeouts().pageLoadTimeout(Duration.ofMinutes(1));
```
Comment (pedroigor) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationInvitationLinkTest.java:77]: Is it because of the time to send the email ? Or can we remove it?

**Review by edewit — APPROVED** [review]

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationInvitationLinkTest.java (hunk 3 above)
Comment (vramik) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationInvitationLinkTest.java:77]: It was originally there: https://github.com/keycloak/keycloak/pull/33334/files#diff-a115c6bafd74ff9e814929582c05e1d3257867062b810c155946449bcb70bf79L227 and I found out that it timeouted few times when I was testing it locally, that's why I increased it to 60s and didn't revert it back. Would you prefer to revert it back?

Comment (pedroigor) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationInvitationLinkTest.java:77]: Works for me. Tks.
